bundle/Controller/Admin/AdminController.php

```diff
@@ -271,10 +293,87 @@ public function handleCollectionAction(Request $request): RedirectResponse
         return $this->redirectToRoute('netgen_information_collection.route.admin.view', ['collectionId' => $collectionId]);
     }
 
+    /**
+     * @throws NonUniqueResultException
+     */
+    public function editAction(Request $request, int $collectionId): RedirectResponse|Response
+    {
+        $this->checkEditPermissions();
+
+        $collection = $this->service->getCollection(new CollectionId($collectionId));
+
+        $location = $collection->getContent()->contentInfo->getMainLocation();
```

Comment:
Make sure that `$location` is not null.

Reply:
BTW, is there a reason we are using the Location here, if Content would also be enough?

Reply:
fixed in commit 2f9f33f6

Reply:
the createUpdateFormForLocation method uses the location when instantiating DataWrapper